commit 9db6a82fa0c6cf0366b20a27555a8f900e0e42d5 (refs/changes/29/4829/4)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2018-09-12T18:44:25+00:00 (1 year, 1 month ago)
    
    TRITON-755 fix unpublished peg.js generated files
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 2f07822..c3dfbf7 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 (nothing yet)
 
+## 1.2.1
+
+- TRITON-755: Fix missing peg.js generated files
+
 ## 1.2.0
 
 - TRITON-755: Add support for 'triton.cmon.groups' tag. This tag is used for
diff --git a/package.json b/package.json
index 779261c..be9d1e0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton-tags",
   "description": "Triton tags parsing and validation",
-  "version": "1.2.0",
+  "version": "1.2.1",
   "author": "Joyent (joyent.com)",
   "dependencies": {
     "assert-plus": "^1.0.0"
